calicoctl/calico_ctl/node.py

```diff
@@ -297,26 +274,6 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
     # Warn if this hostname conflicts with an existing host
     warn_if_hostname_conflict(ip)
 
-    # Install Kubernetes plugin
-    if kubernetes_version:
-        # Build a URL based on the provided Kubernetes_version.
-        url = KUBERNETES_BINARY_URL % kubernetes_version
-        try:
-            # Attempt to install to the default Kubernetes directory
-            install_plugin(KUBERNETES_PLUGIN_DIR, url)
-        except OSError:
-            # Use the backup directory
-            install_plugin(KUBERNETES_PLUGIN_DIR_BACKUP, url)
-
-    # Install rkt plugin
-    if rkt:
-        try:
-            # Attempt to install to the default rkt directory
-            install_plugin(RKT_PLUGIN_DIR, RKT_BINARY_URL)
-        except OSError:
-            # Use the backup directory
-            install_plugin(RKT_PLUGIN_DIR_BACKUP, RKT_BINARY_URL)
```

Comment:
Should we be removing the entire `install_plugin` function?  I don't see it used anywhere else.